opticalFlow/farnebackOpticalFlow.py

```diff
@@ -5,17 +5,43 @@
 Right now, in the result visualization, the intensity of a pixel's motion will change both it's color and magnitude.
 Brighter pixels have more motion.
 The output visualization is stored in the same location as the input video with the name <input_vid_filename>_FB_FLOW.mp4
-
 The idea is that perhaps the data about how certain pixels/features are moving across the screen could be used to figure out how the player camera / aim was changing.
 """
 
+from tkinter import *
+from tkinter import filedialog

```

Comment:
Why do you import all and then this separately?
Can you not just use tkinter.filedialog? Do you even need the import *?

Reply:
Hey, sorry about that, I got lazy about my package imports.
All we need for the GUI file selector to work properly is: `from tkinter import (Tk, Button, filedialog)` and `from tkinter.messagebox import showinfo`. I can update the code and generate and new pull request later today when I have some time if you're cool with that.

Thanks for the review.

Reply:
Fix the importy's and sould be good to go

Reply:
Sounds good, I fixed the package imports. Everything should be set now.